tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -421,8 +421,17 @@ RHODS Notification Drawer Should Contain
     [Documentation]    Verifies RHODS Notifications contains given Message
     [Arguments]     ${message}
     Click Element    xpath=//*[contains(@class,'notification-badge')]
-    Page Should Contain Element
-    ...    xpath=//*[contains(text(),'${message}')]
+    Page Should Contain  text=${message}
+    Click Element    xpath=//*[contains(@class,'notification-badge')]
+
+Clear RHODS Notifications
+    [Documentation]     Clears Notifications present in RHODS dashboard
+    Click Element    xpath=//*[contains(@class,'notification-badge')]
+    Sleep  20s  reason=To avoid Element Not Interactable Exception
```

Comment:
Could you try using **Wait Until Element Is Visible** or **Wait Until Element Is Enabled** insted of sleep for 20min. also could you try with less sleep time

Reply:
Updated